src/pages/api/web3mail/fund-release.tsx

```diff
@@ -131,22 +132,23 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
       );
 
       try {
-        // @dev: This function needs to be throwable to avoid persisting the entity in the DB if the email is not sent
-        await sendMailToAddresses(
-          `
-          Hi ${receiverHandle} !
-          
-          Funds ${action} for the service - ${payment.service.description?.title}`,
+        const email = renderWeb3mail(
+          `Funds ${action} for the service - ${payment.service.description?.title}`,
```

Comment:
it would be too big as title:
- just keep Funds release
- remove the title here and have it in the text